README.md

```diff
@@ -5,15 +5,44 @@
 [![codecov](https://codecov.io/gh/invenia/DeferredFutures.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/invenia/DeferredFutures.jl)
 
 A `DeferredFuture` is like a regular Julia `Future`, but is initialized when `put!` is called on it.
-This means that the data in the `DeferredFuture` lives with the process the data was created on. 
+This means that the data in the `DeferredFuture` lives with the process the data was created on.
 The process the `DeferredFuture` itself lives on never needs to fetch the data to its process.
 This is useful when there is a lightweight controller process which handles scheduling work on and transferring data between multiple machines.
 
 ## Usage
 
 Use a `DeferredFuture` as you would a `Future`.
+```julia
+julia> DeferredFuture()
+DeferredFutures.DeferredFuture(Future(1,1,3,Nullable{Any}()))
 
-Note that `DeferredFuture(n)` does not control where the data lives, only where the `Future` which refers to the data lives. 
+julia> DeferredFuture(3)
+DeferredFutures.DeferredFuture(Future(3,1,4,Nullable{Any}()))
+```
+
+You can also use a `DeferredChannel` as you would a `RemoteChannel`.
+```julia
+julia> DeferredChannel(()->Channel{Int}(10), 4)
+DeferredFutures.DeferredChannel(Future(4,1,5,Nullable{Any}()),#3)
+
+julia> DeferredChannel(4)
+DeferredFutures.DeferredChannel(Future(4,1,6,Nullable{Any}()),DeferredFutures.#2)
+
+julia> DeferredChannel(4, 128; content=Int)
+DeferredFutures.DeferredChannel(Future(4,1,2,Nullable{Any}()),DeferredFutures.#2)
+```
+Note that `DeferredChannel()` will create a `RemoteChannel` with `RemoteChannel(()->Channel{Any}(32), myid())` by default.
```

Comment:
Yeah, but this always seemed like a bad API choice given that by default `Channel()` returns `Channel{Any}(32)`. Since it seems odd you'd want to create a `RemoteChannel` of size 1, I opted for the behaviour of regular `Channel`s.